src/apps/loyalty/containers/inquiries/index.tsx

```diff
@@ -59,42 +65,76 @@ export class Inquiries extends React.Component<RelayProps, State> {
 
     const edges = this.props.user.artwork_inquiries_connection.edges || []
     return edges.map(edge => {
-      // TODO: swap id with __id
-      const { id, artwork } = edge.node
+      const { id, artwork, impulse_conversation_id } = edge.node
       return (
         <Col>
           <InquiryContainer key={id}>
-            <Artwork artwork={artwork as any} />
+            <Artwork
+              artwork={artwork as any}
+              onSelect={this.onArtworkSelected.bind(this, impulse_conversation_id)}
+            />
           </InquiryContainer>
         </Col>
       )
     })
   }
 
+  onArtworkSelected(conversationId: string, selected: boolean) {
```

Comment:
This is actually just a regular callback since I'm not forwarding the `React.FormEvent` in this case, but we add that to the Artwork component though. Good point